README.md

```diff
@@ -22,6 +22,39 @@ vagrant plugin install vagrant-vbguest
 vagrant up
 ```
 
+## Troubleshooting (Mac OS X)
+
+#### Error on box image download
+If you get an error on downloading `debian81.box` image file, then go to
+https://github.com/korekontrol/packer-debian8/releases/download/1.1/debian81.box
+and download it manually, than run command:
+
+```
+vagrant box add /path/to/downloaded/image/debian81.box --name debian81
+vagrant up
+```
+
+#### NFS is reporting that your exports file is invalid
+
+> This may happend if you have previous VMs created and not removed properly or even if you share the computer with someone else who has VMs installed.
+
+```
+sudo rm /etc/exports
```

Comment:
@marek-obuchowicz I tested the command and I get some errors in MAC
running this seems to be ok
```
sudo sed '/VAGRANT-BEGIN/,/VAGRANT-END/d' /etc/exports > /etc/exports
```
when I run with `-i` option I get this error:
> sed: 1: "exports": invalid command code e

So, which variant should be in readme file ? the one with `-i` or the one without it ?
I copied the file in /tmp directory to test it and not delete the real file